reference/segments.v3.yml

```diff
@@ -207,7 +213,7 @@ paths:
         - Shopper Profiles for a Segment
       description: |-
         Add *Shopper Profiles* to a specific *Segment*. 
-        A maximum number of *Shopper Profiles* per request is `50`, and up to three concurrent requests are allowed.
+        A maximum number of *Shopper Profiles* per request is `50`, and up to 10 concurrent requests are allowed.

```

Comment:
Maybe we can use the `Limits` sub-header format here as well for consistency? 🤔 
https://developer.bigcommerce.com/docs/rest-management/customers#update-customers